### ISSUE TITLE
howto example doesn't run on macOS

### ISSUE THREAD

**Issue by jrmuizel** [grovesNL/glow#233]
```
$ cargo run
    Finished dev [unoptimized + debuginfo] target(s) in 0.25s
     Running `/Users/jrmuizel/src/glow/target/debug/howto`
thread 'main' panicked at 'ERROR: 0:1: '' :  version '130' is not supported
ERROR: 0:2: '' :  #version required and missing.
', examples/howto/src/main.rs:86:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by grovesNL** [comment]
Thanks! We could probably default to OpenGL 3.2 or higher on macOS and bump the version in the shader.

**Comment by TheCodeTherapy** [comment]
@grovesNL [pull/238](https://github.com/grovesNL/glow/pull/238) tested on MacBook Pro M1 Max with macOS Ventura 13.0

**Comment by grovesNL** [comment]
Fixed by #238
